modules/statics/src/coins.ts

```diff
@@ -16523,6 +16525,26 @@ export const coins = CoinMap.fromCoins([
     'xrpl.org',
     UnderlyingAsset['txrp:tst-rP9jPyP5kyvFRb6ZiRghAGw5u8SGAmU4bd']
   ),
+  suiToken(
+    'f26941b7-1110-4aa7-a2bc-29807297a51c',
+    'sui:deep',
+    'Deepbook',
```

Comment:
nit
```suggestion
    'DeepBook',
```

Reply:
I added this based on what was in the ticket, can update with the next PR

Reply:
Took reference from the [Deep token code](https://suivision.xyz/package/0xdeeb7a4662eec9f2f3def03fb937a663dddaa2e215b8078a284d026b7946c270?tab=Code):
<img width="1200" alt="image" src="https://github.com/user-attachments/assets/9c725e9b-7db6-4265-a549-c11ef0e6f70f">